lib/Rex/Commands/Sysctl.pm

```diff
@@ -79,6 +94,11 @@ sub sysctl {
       Rex::Logger::debug("$key has already value $val");
     }
 
+    if ( exists $options{persistent} && $options{persistent} eq "true" ) {
```

Comment:
@rapenne-s maybe we can just do `if ( exists $options{persistent} && $options{persistent} ) {` there, so people can also use `persistent => 1` to activate this.